jdaviz/core/template_mixin.py

```diff
@@ -1439,6 +1441,8 @@ def __init__(self, plugin, items, selected, viewer,
         default_mode : str, optional
             What mode to use when making the default selection.  Valid options: first, default_text,
             empty.
+        sort_by : str, optional
+            How to sort the ordering of items.  Valid options: zorder, icon
```

Comment:
Could you explain what the `icon` sorting behavior is here? Zorder is probably pretty self-explanatory but it's not immediately clear to me what the other option is.

Reply:
I'll add it in the docstring as well - but technically its alphabetical ordering of the icons, which effectively means order in which the layers were first created and assigned their icons.